cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/ticket/accesstoken/DefaultAccessTokenFactory.java

```diff
@@ -29,7 +29,7 @@
     /** Default instance for the ticket id generator. */
     @NotNull
     @Autowired(required = false)
-    @Qualifier("accessTokenIdGenerator")
+    @Qualifier("refreshTokenIdGenerator")
```

Comment:
You're in `DefaultAccessTokenFactory`. Why is a `refreshTokenIdGenerator` referenced here?